distribution/pom.xml

```diff
@@ -456,6 +456,10 @@
                                         <argument>org.apache.druid.extensions.contrib:druid-deltalake-extensions</argument>
                                         <argument>-c</argument>
                                         <argument>org.apache.druid.extensions.contrib:druid-spectator-histogram</argument>
+                                        <argument>-c</argument>
+                                        <argument>org.apache.druid.extensions:druid-iceberg-extensions</argument>
```

Comment:
This iceberg extension dependency can be removed as it's a duplicate of line 456. The group id for iceberg extension was recently renamed to `extensions.contrib`, so it probably got mixed up in the merge